StdUi.lua

```diff
@@ -295,3 +295,55 @@ function StdUi:MakeResizable(frame, direction)
 	end)
 end
 
+--- Get a StdUi font object.
+--- This method generates font objects on demand at runtime and caches them for reuse.
+--- Each font is classified by a class and a color code.
+--- Classes are defined by the application via StdUi.config.fontClasses{}. The default class is 'default'.
+--- Color codes are defined by the application via StdUi.config.font.color{}. The default color is 'normal'.
+--- @usage font = StdUi:GetFontObject('scrollingMessage', 'normal')
+--- @usage font = StdUi:GetFontObject('default', 'highlight')
+--- @usage font = StdUi:GetFontObject('default', 'disabled')
+--- @param class string A key of StdUi.config.fontClasses, or 'default'
+--- @param colorCode string A key of StdUi.config.font.color ('normal' by default)
+function StdUi:GetFontObject(class, colorCode)
+	class = class or 'default'
```

Comment:
StdUi coding convention is having semicolons ; at the end of the lines even if lua doesn't require it. Code has to be consistent